boot/assets.go

```diff
@@ -175,7 +175,7 @@ var ErrObserverNotApplicable = errors.New("observer not applicable")
 // for use during installation of the run mode system to track trusted and
 // control managed assets, provided the device model indicates this might be
 // needed. Otherwise, nil and ErrObserverNotApplicable is returned.
-func TrustedAssetsInstallObserverForModel(model *asserts.Model, gadgetDir string, useEncryption bool) (*TrustedAssetsInstallObserver, error) {
+func TrustedAssetsInstallObserverForModel(model *asserts.Model, gadgetDir string, useEncryption bool) (TrustedAssetsInstallObserver, error) {
```

Comment:
I guess that at some point we should rename this to something like `BootAssetsInstallObserver`. Not in this PR of course :slightly_smiling_face: